### PR TITLE
feat: rembadger management

### PR DESCRIPTION
**Pull request by sajanrajdev**
Tackles #1461

### CODE REVIEW

**Review by rayeaster — APPROVED** [review]
LGTM to setup final emission for remBadger by sweep from 2023 version and prepare topup script

File: scripts/issue/1461/rembadger_management.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        == prev_balance + prev_rembadger_dripper_balance + prev_tree_dripper_balance
+    )
+
+    # Transfer BADGER to the treasury ops
+    badger.transfer(
+        r.badger_wallets.treasury_ops_multisig, badger.balanceOf(safe.account)
+    )
+
+    # Cancel upkeeps on both drippers
+    upkeep_manager.cancelMemberUpkeep(rembadger_2023.address)
```
Comment (petrovska-petro) [inline scripts/issue/1461/rembadger_management.py:54]: once this upkeep ids are cancelled, there should be another method to be able to call [withdrawLinkFundsAndRemoveMember](https://etherscan.io/address/0x4c02f0160dc0387b13bcb5e1728c780649e109ac#code#F16#L199) to retrieve $link funds retrieval and removed the ids from storage completely, needs to wait >50 blocks to be able to call it

**Review by petrovska-petro — COMMENTED** [review]
rest looks good:

- confirmed emissions figured 800k vs the official bip and outstanding amount calc in [gsheet](https://docs.google.com/spreadsheets/d/1XjQmMqYozNBc8NekapFxVfiKvMhsEnCWYrtaK6eUH7Y/edit#gid=1189940560) ✅ 
- new emissions schedule for `rembadger` amount, starting and end timestamp looks correct ✅ 
- 5w missing to finalised the defined program by the bip for compound/direct transfer to vault, its operated as 5 direct transfer tx to the vault ✅ 


Screenshot 2024-01-03 at 10 16 29

File: scripts/issue/1461/rembadger_management.py (hunk 1 above)
Comment (sajanrajdev) [reply scripts/issue/1461/rembadger_management.py:54]: Nice catch, thanks. Pushed on https://github.com/Badger-Finance/badger-multisig/pull/1476/commits/dac5d2a734bab72abf276ad45cd0818267f4bd79

**Review by rayeaster — APPROVED** [review]
LGTM with $LINK retrieval for cancelled upkeep

**Review by petrovska-petro — APPROVED** [review]
$link retrieval methods being added in latest commit, g2 be posted ✅

**Pull request merged**